src/prometheus.rs

```diff
@@ -0,0 +1,17 @@
+pub const METRIC_NAME_LABEL: &str = "__name__";
+
+pub mod proto {
+    include!(concat!(env!("OUT_DIR"), "/prometheus.rs"));
+}
+
+#[cfg(feature = "sinks-prometheus")]
```

Comment:
Nit: if these are only used by the sink, should they just be defined there instead?

Reply:
Probably. I had them there before, but thought they would end up being used by both. I'll move them back.